Source/CarthageKit/Xcode.swift

```diff
@@ -878,11 +879,30 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 
 		return schemeSignals
+			.catch { error in
+				switch (dependency, error.code) {
+				case (.GitHub(let repo), CarthageErrorCode.NoSharedSchemes.rawValue):
+					return .error(addIssueFilingSuggestionToError(error, repo))
+				default:
+					return .error(error)
+				}
+			}
 	}
 
 	return (buildOutput, copyProducts)
 }
 
+/// Augments a no shared schemes error description to include a suggestion
+/// to file a GitHub issue in GitHub repositories.
+///
+/// Returns a no shared schemes error with an issue filing suggestion.
+private func addIssueFilingSuggestionToError(error: NSError, repo: GitHubRepository) -> NSError {
+	let msg = error.localizedDescription + "\n\nIf you believe this to be an error, please file an issue with the maintainers at \(repo.newIssueURL.absoluteString!)"
```

Comment:
What does the URL look like printed? (I would expect it'd be the same as `absoluteString`, but maybe not.)